lib/Guides/Compiler/GlobMenuFixerTransformer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Guides\Compiler;
+
+use phpDocumentor\Guides\Compiler\CompilerContext;
+use phpDocumentor\Guides\Compiler\NodeTransformer;
+use phpDocumentor\Guides\Nodes\Menu\GlobMenuEntryNode;
+use phpDocumentor\Guides\Nodes\Node;
+
+/**
+ * This class will fix the glob menu entries.
+ *
+ * The glob menu entries are not correctly added in the docs.
+ * This transformer will fix the glob menu entries by replacing the '*' with '**'. This add a certain
+ * risk if the user had the intention to match only documents in the current directory. But this can be solved
+ * adding a `/` at the beginning of the glob pattern.
+ *
+ * @implements NodeTransformer<GlobMenuEntryNode>
+ */
+final class GlobMenuFixerTransformer implements NodeTransformer
```

Comment:
If I understand correctly, we are not following the rst specification properly in some parts of the documentation. If that's the case, it would be nice to have a logging mechanism to this so that we ultimately fix the broken docs and remove this transformer. I think the same could also be done for the thing you did yesterday about the syntax of `section-author`

Reply:
Question is, are you going to fix all versions in history?

Reply:
Oh right, I missed that very important point!

Reply:
That is true, but I started to drop the rendering of some older versions (currently [DBAL](https://github.com/doctrine/doctrine-website/blob/master/config/data/projects.yml#L24) and [ORM](https://github.com/doctrine/doctrine-website/blob/master/config/data/projects.yml#L36) only), so we could reach a point in the projects' history where all issues can be fixed.